lib/browser.js

```diff
@@ -123,6 +123,17 @@ exports.utils = require('./utils.js');
  */
 exports.Document = DocumentProvider();
 
+/**
+ * function stub for model
+ *
+ * @method model
+ * @api public
+ * @return {object} an empty object
+ */
+exports.model = function() {
+  return {};
```

Comment:
Why not `return null` or something like that? I'm hesitant to just return an empty object because then it will have an inconsistent and incompatible return type vs `model()` in server land.

Reply:
Yes, good idea, done